packages/react-composites/src/composites/ChatComposite/file-sharing/FileCard.tsx

```diff
@@ -49,7 +49,7 @@ export const FileCard = (props: FileCardProps): JSX.Element => {
   const { fileName, fileExtension, progress, actionIcon } = props;
   const theme = useTheme();
 
-  const showProgressIndicator = (): boolean => progress !== undefined && progress < 1;
+  const showProgressIndicator = (): boolean => progress !== undefined && progress < 1 && progress > 0;
```

Comment:
nit.

Can we swap the order to be progress > 0 && progress < 1 so we can read it  left to right that we want to show progress from 0 -> 1?